odoo/addons/base/res/res_config.py

```diff
@@ -437,14 +441,15 @@ def _get_classified_fields(self):
                 {   'default': [('default_foo', 'model', 'foo'), ...],
                     'group':   [('group_bar', [browse_group], browse_implied_group), ...],
                     'module':  [('module_baz', browse_module), ...],
+                    'config':  [('config_plop', config_parameter), ...],
                     'other':   ['other_field', ...],
                 }
         """
         IrModule = self.env['ir.module.module']
         Groups = self.env['res.groups']
         ref = self.env.ref
 
-        defaults, groups, modules, others = [], [], [], []
+        defaults, groups, modules, config, others = [], [], [], [], []
```

Comment:
Naming convention: all lists are plural except for `config`; what about `configs`?

Reply:
If that makes you happy, why not.

Reply:
fixed :)